ui/components/multichain/notification-detail-block-explorer-button/notification-detail-block-explorer-button.tsx

```diff
@@ -36,7 +36,11 @@ export const NotificationDetailBlockExplorerButton = ({
   }, [defaultNetworks]);
 
   const blockExplorerUrl =
-    defaultNetwork?.rpcPrefs?.blockExplorerUrl ?? nativeBlockExplorerUrl;
+    defaultNetwork?.rpcPrefs?.blockExplorerUrl ?? blockExplorerConfig?.url;
+
+  const blockExplorerButtonText = blockExplorerConfig?.name
```

Comment:
This name might be incorrect if `defaultNetwork?.rpcPrefs?.blockExplorerUrl` was used (e.g. if the user has configured their own block explorer locally and we chose to use that, we shouldn't use a name from a different block explorer here).

Reply:
You are absolutely right. Fixed. Thx!